index.md

```diff
@@ -4,3 +4,11 @@ layout: default
 
 {% include 01-name.md %}
 
+<br>
+
+{% include 02-image.md %}
```

Comment:
![Image of Yaktocat](https://octodex.github.com/images/yaktocat.png).

Developing new system and build different website